sql/setup.sql

```diff
@@ -1,2 +1,9 @@
 -- Use this file to define your SQL tables
 -- The SQL in this file will be executed when you run `npm run setup-db`
+DROP TABLE IF EXISTS onepiece;
+
+CREATE TABLE onepiece (
+    id BIGINT GENERATED ALWAYS AS IDENTITY,
```

Comment:
missing a couple of things here. `BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY`